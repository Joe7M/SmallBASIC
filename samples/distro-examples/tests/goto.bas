goto TestLabel

print "GOTO failed"

LABEL TestLabel

REM Don't crash when label does not exist
goto 1
