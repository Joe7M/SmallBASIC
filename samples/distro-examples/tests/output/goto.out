

 * COMP-ERROR AT Main:2 * 
Description:
Label '1' is not defined

